### ISSUE TITLE
ARM64 deployment fails

### ISSUE THREAD

**Issue by sysedwinistrator** [kadalu/kadalu#578]
Hi there,
I am trying to deploy kadalu on k8s for the first time.
On my first attempt, I wasn't aware of the arm GitHub branch and Docker tag. I used the generic yaml , but the operator container wouldn't start. (I didn't look at the logs, sorry). So I modified the yaml and changed the the Docker tag from devel to 0.8.2, and the operator started. Now, however, the csi-nodeplugin and csi-provisioner containers wouldn't start. The containers failed with `exec format error` (which means the wrong architecture was downloaded) like in #264 and #271.
After finding these issues, I used the yaml from the arm branch (which uses the arm tag container image). Now the operator wouldn't start, so I looked into the logs:
```
Traceback (most recent call last):
  File "/kadalu/start.py", line 3, in <module>
    from kadalulib import Monitor, Proc, logging_setup
  File "/kadalu/kadalulib.py", line 11, in <module>
    import xxhash
ModuleNotFoundError: No module named 'xxhash'
```
This has already been reported in #474.

I also tried using 0.7.7. But I still get `exec format error` on csi-provisioner and csi-nodeplugin.

**Comment by amarts** [comment]
Hi @sysedwinistrator... We fixed few issues seen with ARM64 deployment after `0.8.2` release. The latest pre release is working `0.8.3-alpha2`. Please try that and let us know if it works. We are planning to make the release of 0.8.3 soon. If you confirm it works, then we will do it this week itself.

**Comment by sysedwinistrator** [comment]
Hi @amarts, thank you for your reply. As I see, you have already released `0.8.3` to docker.
I realised that the mistake was on my side: I only changed the tag of the kadalu-operator image, but didn't also change the `KADALU_VERSION` in the yaml. Sorry for the trouble.
After I put version `0.8.3` in both places, the operator started, but not the csi-provisioner and csi-nodeplugin; they failed with `ImageInspectError`. Something was pulling an unqualified image (busybox), so I had to add docker.io as registry for unqualified images (I'm using cri-o). Kadalu is now up and running!

I found the unqualified pull in `templates/csi.yaml.j2`. `busybox` and `alpine/socat:1.0.5` are the only unqualified images in that file, all other images use docker.io.
